### PR TITLE
Test the CI process of the application build

### PR DESCRIPTION
**Pull request by JayAgola**
issue  #40 resolve

**Description**:

- This PR resolves issues causing CI tests to fail during automated builds.

- Identified and fixed [mention any specific cause if known, e.g., widget initialization, API stubs].

- Ensured all tests pass locally and in CI environment after changes.

### CODE REVIEW

**Comment by JayAgola** [comment]
@Mehul-Kumar-27 @aspiringgarv  issue solved please review pull request

File: .github/workflows/flutter_ci.yml [hunk 1]
```diff
@@ -31,20 +30,20 @@ jobs:
...
+#      uses: actions/upload-artifact@v4
+#      with:
+#        name: build-android
+#        path: build/app/outputs/flutter-apk/app-release.apk
+#
+#    - name: Archive and upload iOS artifacts
+#      uses: actions/upload-artifact@v4
+#      with:
+#        name: build-ios
+#        path: build/ios/Runner.app
```
[the remark this answers is not in the document]
Comment (JayAgola) [reply .github/workflows/flutter_ci.yml:49]: for  build the app i need keystore file for if you have share to me or i have to create new keystore file

[the remark this answers is not in the document]
Comment (Mehul-Kumar-27) [reply .github/workflows/flutter_ci.yml:49]: I don't plan to publish it on the app store, or Google play. Rather just make a release on GitHub.

**Comment by JayAgola** [comment]
@Mehul-Kumar-27 , I've made this test case compatible for the Android APK, but for the iOS app, a macOS is required. To proceed with the iOS app, please create a new issue for it.

**Comment by JeetJani-11** [comment]
@JayAgola Please make a Google Doc with your GitHub username as the document's name in this folder(https://drive.google.com/drive/folders/1qWd_Vxv6G2aNNPbPtTNpeulYovOV-cMi). Use it to keep track of your daily contributions so we can monitor your progress.

<blockquote><div><strong><a href="https://drive.google.com/drive/folders/1qWd_Vxv6G2aNNPbPtTNpeulYovOV-cMi">2024 - Google Drive</a></strong></div></blockquote>

**Pull request merged**